### PR TITLE
webOS-ify

### PR DESCRIPTION
**Pull request by NoelMacwan**

### CODE REVIEW

**Comment by morphis** [comment]
Can you submit the same PR the for wop-11.0 branch too?

File: README.mkdn [hunk 1]
```diff
@@ -1,50 +1,18 @@
-CyanogenMod
+webOS
 ===========
 
-Submitting Patches
```
Comment (morphis) [inline README.mkdn:4]: We should add a notice here that we're accepting pull-requests on github.com/webOS-ports for the repositories we've forked.

**Comment by morphis** [comment]
Ah you did already the PR for wop-11.0. Forget abou the comment above then :)

**Comment by NoelMacwan** [comment]
I have sent already, please check in wop 11.0 branch
Yeah, that will be very cool or else people will have doubts that whether
you guys accept pull request or not..

**Comment by NoelMacwan** [comment]
so when you will accept this ?

**Comment by NoelMacwan** [comment]
New PR sent !

**Pull request closed without merging**